src/Components/CreateImageWizardV2/steps/Details/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+import {
+  Form,
+  FormGroup,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  Text,
+  Title,
+} from '@patternfly/react-core';
+
+import { ValidatedTextInput } from '../../ValidatedTextInput';
+import {
+  isBlueprintDescriptionValid,
+  isBlueprintNameValid,
+} from '../../validators';
+
+const DetailsStep = () => {
+  const [name, setName] = React.useState('');
+  const [description, setDescription] = React.useState('');
+
+  const handleNameChange = (
+    _event: React.FormEvent<HTMLInputElement>,
+    name: string
+  ) => {
+    setName(name);
+  };
+  const handleDescriptionChange = (
+    _event: React.FormEvent<HTMLInputElement>,
+    name: string
+  ) => {
+    setDescription(name);
+  };
+  return (
+    <Form>
+      <Title headingLevel="h2">Details</Title>
+      <Text>
+        Optionally enter a name to identify your image later quickly. If you do
+        not provide one, the UUID will be used as the name.
+      </Text>
+      <FormGroup label="Blueprint Name" fieldId="blueprint-name">
```

Comment:
Let's use sentence case for the FormGroup labels :)

```suggestion
      <FormGroup label="Blueprint name" fieldId="blueprint-name">
```